starwars/src/App.js

```diff
@@ -30,11 +29,14 @@ class App extends Component {
     };
 
     render() {
-        return ( <
-            div className = "App" >
+        return ( < div className = "App" >
             <
-            h1 className = "Header" > React Wars < /h1> < /
-            div >
+            h1 className = "Header" > React Wars < /h1>     <
+            CharacterList characters = { this.state.starwarsChars }
+            cars = { this.state.starwarsChars.vehicles }
+            />
+            console.log(this.state.starwarsChars.vehicles) <
```

Comment:
Put curly braces around any javascript you want executed in the render section of your code.  Here, you would instead have ```{console.log(this.state.starwarsChars.vehicles)}```.